domains/Discussions/Tests/Feature/QuestionsGetAnswersTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Domains\Discussions\Tests\Feature;
+
+use Carbon\Carbon;
+use Faker\Factory;
+use Tests\TestCase;
+use Faker\Generator;
+use Illuminate\Http\Response;
+use Domains\Accounts\Models\User;
+use Domains\Discussions\Models\Answer;
+use Domains\Discussions\Models\Question;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Discussions\Database\Factories\AnswerFactory;
+use Domains\Discussions\Database\Factories\QuestionFactory;
+
+class QuestionsGetAnswersTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    private User $user;
+    private User $secondUser;
+    private Question $question;
+    private Answer $answer;
+    private Answer $secondAnswer;
+    private Answer $thirdAnswer;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->user = UserFactory::new()->create();
+        $this->secondUser = UserFactory::new()->create();
+
+        $this->question = QuestionFactory::new([
+            'author_id' => $this->user->id
+        ])->create();
+
+        $this->answer = AnswerFactory::new([
+            'question_id' => $this->question->id,
+            'author_id' => $this->user->id,
+            'created_at' => Carbon::now()->subWeek()->toDateTimeString()
+        ])->create();
+
+        $this->secondAnswer = AnswerFactory::new([
+            'question_id' => $this->question->id,
+            'author_id' => $this->secondUser->id,
+            'created_at' => Carbon::now()->toDateTimeString()
+        ])->create();
+
+        $this->thirdAnswer = AnswerFactory::new([
+            'question_id' => $this->question->id,
+            'author_id' => $this->user->id,
+            'created_at' => Carbon::now()->subWeek()->toDateTimeString()
+        ])->create();
```

Comment:
Do we need to create all these at once? We should only create resources in this method if they'll be used by the majority of test cases in this class. Slight variations or specific resources' creation should be done in their own test case.

Reply:
Good point! From what I see, i can remove the `thirdAnswer` since it's only use in one test, but to me the other variables are used throughout the tests and so should be left. Do you agree?

Reply:
It looks ok to me, yes.